go/vm/lfvm/ct_test.go

```diff
@@ -374,6 +375,7 @@ func BenchmarkLfvmStackToCtStack(b *testing.B) {
 		stack.pushEmpty().SetUint64(uint64(i))
 	}
 	for i := 0; i < b.N; i++ {
-		convertLfvmStackToCtStack(stack)
+		ctStack := st.NewStack()
```

Comment:
nit: maybe for the benchmark reusing the same stack every time is the better choice, since it is closer to the actual use case scenario.